var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -110,6 +114,10 @@ def make_defs(self):
                 'TARGET=PILEDRIVER',
                 'TARGET=ARMV8'
             ]
+        if self.spec.variants['cpu_target'].value:
```

Comment:
Mh I'm not too sure. But doing a short test, it seems as if only the last mention of a variable is honored at all:
```
❯ cat Makefile
print-%  : ; @echo $* = $($*)

❯ make TARGET=1 TARGET=2 print-TARGET
TARGET = 2
```
This would mean that only ```'TARGET'=ARMV8'``` has any effect. I'm unsure what should be the correct behavior here.

I'm unsure what ```spack.architecture.sys_type()``` is.
- If it's something that is automatically detected, then it shouldn't be necessary at all, as this is what produced our problems in the first place (we build on backend-nodes with newer CPUs and the resulting build didn't work on the frontends due to illegal instructions).
- If it's essentially a cross-compiling flag, that can be chosen, I'd prefer a one-size-fits-all-solution

cc @kljohann